tests/unit/components/DocumentationTopic.spec.js

```diff
@@ -507,6 +507,27 @@ describe('DocumentationTopic', () => {
     expect(wrapper.find(ViewMore).exists()).toBe(false);
   });
 
+  it('renders `ViewMore` with correct language path', () => {
+    wrapper.setProps({
+      enableMinimized: true,
+      swiftPath: null,
+      objcPath: 'documentation/objc',
+      interfaceLanguage: 'occ',
+    });
+    const objcViewMore = wrapper.find(ViewMore);
+    expect(objcViewMore.exists()).toBe(true);
+    expect(objcViewMore.props('url')).toEqual('/documentation/objc'); // normalized path
+
+    wrapper.setProps({
+      objcPath: null,
+      swiftPath: 'documentation/swift',
+      interfaceLanguage: 'swift',
+    });
+    const swiftViewMore = wrapper.find(ViewMore);
+    expect(swiftViewMore.exists()).toBe(true);
+    expect(swiftViewMore.props('url')).toEqual('/documentation/swift'); // normalized path
```

Comment:
Looks good, thanks. You may want to add one more additional scenario for the page that has both a Swift/ObjC path since there is also logic for that scenario.

Reply:
https://github.com/apple/swift-docc-render/blob/b8ba1d34ec705c2ce4dfa4916dc37ee4945d1847/tests/unit/components/DocumentationTopic.spec.js#L493

This test should be testing the scenario where we have both paths & by default we have `interfaceLanguage` set to `swift`. Do you think that's enough?

Reply:
I think that's ok. It just might be good to have one additional one for the scenario where both variants are available and you're on the ObjC one, since that would trigger your logic in this PR for the query param.